package.json

```diff
@@ -8,6 +8,9 @@
   "devDependencies": {
     "@commitlint/cli": "^17.6.7",
     "@commitlint/config-conventional": "^17.6.7",
+    "@tsconfig/node16": "^16.1.1",
+    "@types/node": "^20.5.9",
```

Comment:
noting

https://raw.githubusercontent.com/tsconfig/bases/main/bases/node16.json
- compilerOptions
  - lib: es2021
  - target: es2021

https://github.com/tsconfig/bases/blob/main/bases/node20.json
- compilerOptions
  - lib: es2023
  - target: es2022

Reply:
kinda off-topic but i'll share anyway
https://github.com/tsconfig/bases/blob/main/bases/node12.json
https://github.com/tsconfig/bases/blob/main/bases/node14.json
seemed strange at first seeing their module systems/resolutions in the future (node16) :suspect:
but makes sense since node10 module res only supports cjs, but node16 supports esm too
so the change was done across the board [4mo ago](https://github.com/tsconfig/bases/commit/a8c454f4a1caf4511b1e58a6b71ef4529264f628)

Reply:
it's a minimum compat thing, so since our minimum is node 16 we should use the node16 config.